lib/ring_logger/client.ex

```diff
@@ -2,7 +2,7 @@ defmodule RingLogger.Client do
   use GenServer
 
   @moduledoc """
-  Interact with the RingLogger
+  #Interact with the RingLogger
```

Comment:
I think this was an accidental commit here.

Reply:
yes, I will grab that.